app/communities-config.js

```diff
@@ -57,8 +59,10 @@ const communitiesConfig = {
     // },
   },
   dev: {
-    3: {
+    2: {
       origin: 'http://localhost:31000',
```

Comment:
why origin is localhost with this port?

Reply:
it should just be a random port for the dev environment, isn't it?

Reply:
If it's only for dev purposes looks ok to me, just wondering